stable/postgresql/templates/deployment.yaml

```diff
@@ -138,6 +138,14 @@ spec:
             mountPath: /conf
             readOnly: true
         {{- end }}
+        livenessProbe:
+          httpGet:
+            path: /metrics
+            port: 9187
```

Comment:
can you please use the name of the service port (`metrics`) instead of the hardcoded number?

Reply:
Changed to `port: metrics` in the latest commit.